numpy/ma/tests/test_mrecords.py

```diff
@@ -28,21 +28,14 @@
     )
 
 
-class TestMRecords(TestCase):
-    # Base test class for MaskedArrays.
-    def __init__(self, *args, **kwds):
-        TestCase.__init__(self, *args, **kwds)
-        self.setup()
+class TestMRecords(object):
 
-    def setup(self):
-        # Generic setup
-        ilist = [1, 2, 3, 4, 5]
-        flist = [1.1, 2.2, 3.3, 4.4, 5.5]
-        slist = [b'one', b'two', b'three', b'four', b'five']
-        ddtype = [('a', int), ('b', float), ('c', '|S8')]
-        mask = [0, 1, 0, 0, 1]
-        self.base = ma.array(list(zip(ilist, flist, slist)),
-                             mask=mask, dtype=ddtype)
+    ilist = [1, 2, 3, 4, 5]
+    flist = [1.1, 2.2, 3.3, 4.4, 5.5]
+    slist = [b'one', b'two', b'three', b'four', b'five']
+    ddtype = [('a', int), ('b', float), ('c', '|S8')]
+    mask = [0, 1, 0, 0, 1]
+    base = ma.array(list(zip(ilist, flist, slist)), mask=mask, dtype=ddtype)
```

Comment:
OK, seems good to me, the view test does not have an explicit copy of `base` but it should not matter.